src/plugins/LADTable/components/LADRow.vue

```diff
@@ -110,24 +112,34 @@ export default {
     },
     methods: {
         updateValues(datum) {
-            let newTimestamp = this.formats[this.timestampKey].parse(datum),
-                limit;
-
-            if(this.shouldUpdate(newTimestamp)) {
-                this.timestamp = this.formats[this.timestampKey].parse(datum);
-                this.value = this.formats[this.valueKey].format(datum);
-                limit = this.limitEvaluator.evaluate(datum, this.valueMetadata);
-                if (limit) {
-                    this.valueClass = limit.cssClass;
-                } else {
-                    this.valueClass = '';
+            if(datum) {
+                let newTimestamp = this.getParsedTimestamp(datum),
+                    limit;
+
+                if(this.shouldUpdate(newTimestamp)) {
+                    this.timestamp = this.getParsedTimestamp(datum);
+                    this.value = this.formats[this.valueKey].format(datum);
+                    limit = this.limitEvaluator.evaluate(datum, this.valueMetadata);
+                    if (limit) {
+                        this.valueClass = limit.cssClass;
+                    } else {
+                        this.valueClass = '';
+                    }
                 }
+            } else {
+                this.resetValues();
```

Comment:
Let's reset values [when the user changes bounds](https://github.com/nasa/openmct/pull/3114/files#diff-283f9645a299cbc3994fbf01050dc9eeR161). I think that will simplify things.